src/components/Personalization/dom-actions/dom/helperForShadow.js

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const SHADOW_SEPARATOR = ":shadow";
+
+export const isNotShadowSelector = str => str.indexOf(SHADOW_SEPARATOR) === -1;
```

Comment:
I recommend not using a negative. Instead, make it a positive (`isShadowSelector`) and let the consuming module determine whether to negate it.

Reply:
Here I was also guided by existing code in `helperForEq.js` where we have `isNotEqSelector`. I'll fix it in both places then, per your recommendation..

Reply:
Thank you!